src/main/java/Priority.java

```diff
@@ -0,0 +1,5 @@
+public enum Priority {
+    Minor,
```

Comment:
Let's follow java naming conventions for enums

```
Because they are constants, the names of an enum type's fields are in uppercase letters.
```

Link: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Could you update other constants in other enums appropriately?